cocos/rendering/forward/forward-flow.ts

```diff
@@ -30,6 +30,7 @@ import { ForwardFlowPriority } from '../enum';
 import { ForwardStage } from './forward-stage';
 import { RenderPipeline } from '../render-pipeline';
 import { Camera } from '../../render-scene/scene';
+import { ARBackgroundStage } from '../xr/ar/ar-background-stage';
```

Comment:
I think if we import xr module here, it cannot be culled even if user does not use it.
We need to use `engine/exports/xxx.ts` to control feature culling.

Reply:
Do you want us to write a background stage with an empty implementation? Used for non xr?

Reply:
You might create ARBackgroundStage from `globalThis.__globalXR` and remove `import { ARBackgroundStage } from '../xr/ar/ar-background-stage';`.

Reply:
Switch to an empty implementation might not be possible. If possible, it is also ok.

Reply:
Yes, Let me try the feasibility